src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const styleObject = {};
+
+  stylesArray.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      styleObject[key.trim()] = value.trim();
+    }
```

Comment:
[CHECKLIST ITEM 5] - Avoid using loops in this task, use iteration methods instead. Consider using `map` to transform the array of style strings to an array of `[key, value]` pairs and then `reduce` to build the final object.